### ISSUE TITLE
Unable to update 3rd-party filters in Safari.

### ISSUE THREAD

**Issue by BillyJoeJimBob** [uBlock-LLC/uBlock#1336]
I am unable to update any filters in the third-party filter tab.  Selecting the "Update Now" button causes the contents of the tab to fade out, then nothing happens, and the tab contents are non-responsive.  Switching to another tab and back to the 3rd party filter tab removes the faded/non-responsiveness issue, but the filters will not update beyond April 19.  Selecting or unselecting filters, closing Safari, restarting the computer or disabling all other Safari add-ons/extensions does not fix this issue.  Thank you.  -B

I am using:  OS X 10.10.3/Safari 8.0.5/uBlock 0.9.3.5  

Images of faded, non-responsive tab:

![ublock dashboard 2015-04-29 12-01-14](https://cloud.githubusercontent.com/assets/11702508/7399939/27f05a7c-ee6e-11e4-9361-75c8543f2bee.png

![ublock dashboard 2015-04-29 12-01-59](https://cloud.githubusercontent.com/assets/11702508/7399958/4e3b3422-ee6e-11e4-87d3-5c150975ffde.jpg)
)

**Comment by TSloper** [comment]
I ran into the same issue. I originally installed uBlock from Safari -> Safari Extensions. This version misbehaves. If you uninstall and install it from the uBlock site it will update without any issues.

**Comment by BillyJoeJimBob** [comment]
TSloper

Installing direct from uBlock seemed to fix the issue, thanks.

**Comment by niteowl2** [comment]
I originally installed from the uBlock site and I'm still having this issue.

**Comment by BillyJoeJimBob** [comment]
Niteowl2

After uninstalling uBlock, i cleared Safari caches and cookies before reinstalling, and now it seems to work.  Before that, if I set up uBlock how I wanted and then waited until the filters were eventually updated on their own, it would work if I left it alone. If I changed anything, changing filters, hitting the Purge or Update buttons would cause the filters to revert to April 19 and freeze.  None of these issues occur in FF/Chrome/Opera with the same extensions/filters.
